pkg/costmodel/costmodel.go

```diff
@@ -487,21 +469,29 @@ func (cm *CostModel) ComputeCostData(cli prometheusClient.Client, cp costAnalyze
 				// recreate the key and look up data for this container
 				newKey := NewContainerMetricFromValues(ns, podName, containerName, pod.Spec.NodeName, clusterID).Key()
 
-				RAMReqV, ok := RAMReqMap[newKey]
-				if !ok {
-					klog.V(4).Info("no RAM requests for " + newKey)
-					RAMReqV = []*util.Vector{{}}
+				// k8s.io/apimachinery/pkg/api/resource/amount.go for explanations of
+				// the types that memory/cpu requests are
+				ramRequestBytes := container.Resources.Requests.Memory().Value()
```

Comment:
I'm not sure what in amount.go guarantees this to be in bytes. is it actually resource/quantity.go that shims Value->ScaledValue(0) ? Does using ScaledValue(0) make more sense to ensure this is in bytes? (definitely not trying to make this a leading question, just wondering.)

Reply:
Modified comment for a better explanation. I think using `Value()` is fine because the unit of memory in K8s [is bytes](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-types). It is just a shim to `ScaledValue(0)` but I think using value is clear given the expectation set by general K8s documentation.